_services/automation.html

```diff
@@ -0,0 +1,100 @@
+header-img: "/img/blank-notepad.jpg"
+top-section: "<p>Automation isn’t a one-size-fits-all solution, a tailored strategy will help your business unlock its full potential.</p>
+<p>That’s where our Business Process Automation expertise makes all the difference.</p>"
+---
+
+<div class="flex-row">
+  <section>
+    <div class="h2-wrapper">
+      <img src="/img/build-icon.svg" class="h2-icon" />
+      <h2>Our Services</h2>
+    </div>
+    <p>
+      Whether you’re automating from scratch or looking to refine your existing
+      business automations, our team seamlessly integrates with your business to
+      identify bottlenecks and resolve them through analysis, automations and
+      AI.
+    </p>
+    <p>Our services include:</p>
+    <ul>
+      <li>Business Process Automation</li>
+      <li>Sales Process Automation</li>
+      <li>Marketing Automation</li>
+      <li>Customer Success Optimization</li>
+      <li>CRM Automation</li>
+    </ul>
+    <p>
+      Our automation services focus on key components of your tech stack, and
+      getting them to work together through the use of Automation tools such as
+      Customer Relationship Management (CRM) Systems Pipedrive and HubSpot,
+      Marketing Automation Platforms like HubSpot Marketing Hub, Sales
+      Automation Tools like Pipedrive’s Sales Automation System, Workflow
+      Automation Platforms like Zapier, Email and Communication Automation tools
+      like Pipedrive’s Campaigns feature and HubSpot’s Workflow Automations, and
+      more.
+    </p>
+    <p>
+      By leveraging these advanced technologies we streamline your business
+      processes, reduce manual tasks, and drive sustainable growth—saving you
+      valuable time and resources.
+    </p>
+  </section>
+  <img class="side-image" src="/img/meeting-room-original.jpg" />
+</div>
+
+<div class="flex-row">
+  <img class="side-image" src="/img/business-man-away.jpg" />
+  <section>
+    <div class="h2-wrapper">
+      <img src="/img/explore-icon.svg" class="h2-icon" />
+      <h2>Why choose us</h2>
+    </div>
+    <p>
+      We work directly with you and your teams to understand your businesses
+      unique goals and identify opportunities for automation that can streamline
+      operations and drive growth.
+    </p>
+    <p>
+      We have run, and automated, our own successful companies - we know
+      firsthand the common pitfalls businesses encounter. Our expertise allows
+      us to help you avoid those obstacles through smart automation strategies.
+    </p>
+    <p>
+      We have certified technical expertise in automation platforms such as
+      <a href="/hubspot">HubSpot</a>, <a href="/pipedrive">Pipedrive</a>, and
+      other leading systems. Through these partnerships, we implement custom
```

Comment:
Youve already said this. Add the links above and scrap this paragraph. Repetitive and pushy.